charts/openmetadata/templates/deployment.yaml

```diff
@@ -2,21 +2,31 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "OpenMetadata.fullname" . }}
+  annotations:
+    {{- if .Values.annotations }}
+    {{- range $key, $val := fromYaml .Values.annotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+    {{- end}}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
+    {{- include "OpenMetadata.labels" . | nindent 4 }}  
+
 spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      {{- include "OpenMetadata.selectorLabels" . | nindent 6 }}
+      {{- include "OpenMetadata.labels" . | nindent 6 }} 
+
   template:
     metadata:
       {{- with .Values.podAnnotations }}
```

Comment:
All the Kubernetes Objects should refer to this by calling
`{{- include "OpenMetadata.annotations" . | nindent 8 }}`.

Refactor to _helper.tpl as mentioned above in review.

Reply:
added {{- include "OpenMetadata.annotations" . | nindent 4 }}